src/data/projects.js

```diff
@@ -208,6 +208,14 @@ const projects = [
     website: "https://satnogs.org/",
     image: "SatNOGS.png"
   },
+  {
+    name: "NEODyS-2",
+    dateAdded: "31/08/2023",
+    tags: ["Near Earth Objects", "Asteroids","OSINT"],
```

Comment:
I only noticed after the PR there are specific tags. Maybe astrodynamics, ground control and planetary defence can be added here.